metrics-dropwizard/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'scala'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation project(':usi-metrics')
+    compile group: 'io.dropwizard.metrics', name: 'metrics-core', version: '4.0.2'
+    compile group: 'io.dropwizard.metrics', name: 'metrics-jvm', version: '4.0.2'
+    compile group: 'com.github.vladimir-bukhtoyarov', name: 'rolling-metrics', version:  '2.0.4'
+    compile group: 'org.hdrhistogram', name: 'HdrHistogram', version:  '2.1.10'
+
+    // Common project dependencies
```

Comment:
What do you mean?

Reply:
Oh, thanks for the reminder. I wanted to move them out as dependencies for all projects (logging, akka etc. will be used everywhere)